pkg/controller/deployment/generic_deployment_controller.go

```diff
@@ -22,6 +22,13 @@ import (
 	"github.com/openshift/cert-manager-operator/pkg/operator/operatorclient"
 )
 
+// TODO: This is just a temporary placeholder.
```

Comment:
In this PR, client registration is removed. Am I right to understand, the status can be seen in implementation of CFE-668?

Reply:
Status of deployments (3 controllers from upstream) can be seen in CFE-668.